modules/specialized-guides/pages/public-cloud-guide/faq/faq-appendix.adoc

```diff
@@ -1,6 +1,6 @@
 = Appendix
 
-== Countries that can transact {productname} through the Cloud Marketplace
+== Countries that can transact {susemgr} {payg} through the Cloud Marketplace
```

Comment:
{productname} should remain. This is a global attribute set in our parameters.yml file. See lines 6-8:

```
    asciidoc:
      - attribute: productname
        value: "SUSE Manager"
```